crud/borders.lua

```diff
@@ -163,8 +163,8 @@ end
 -- @param string space_name
 --  A space name
 --
--- @param ?string index_name
---  An index name (by default, primary index is used)
+-- @param ?string index_id
+--  An index identifier (by default, primary index is used)
```

Comment:
Names are inconsistent: space_name, index_id. AFAIS, both accept as space/index string name as well as its numeric ID.

Reply:
`borders.min()` do nothing with arguments and pass them to `get_border`, `get_border` too and calls `call_get_border_on_router` that uses `space_name` as a name of space with type string in its body. The same with `index_name`. I suggest using `{index, space}_name` because using the name is more friendly for humans instead of indices. Additionally document functions arguments and mention that space and index ID's accepted too.

Reply:
Ouch, I missed the `checks()` call. Maybe it worth to allow numerical IDs here?

Generally speaking `box.space[<...>]` accepts both numerical IDs and string names, so upward APIs usualy reflects this property. I would use `*_name` names and comment that a numerical ID is okay here too.

Reply:
Added `numeric` type too:
```lua
--- a/crud/borders.lua
+++ b/crud/borders.lua
@@ -17,7 +17,7 @@ local STAT_FUNC_NAME = '_crud.get_border_on_storage'
 
 
 local function get_border_on_storage(border_name, space_name, index_name, field_names)
-    dev_checks('string', 'string', 'string|number', '?table')
+    dev_checks('string', 'string|number', 'string|number', '?table')
 
     assert(border_name == 'min' or border_name == 'max')
 
@@ -67,7 +67,7 @@ else
 end
 
 local function call_get_border_on_router(border_name, space_name, index_name, opts)
-    checks('string', 'string', '?string|number', {
+    checks('string', 'string|number', '?string|number', {
         timeout = '?number',
         fields = '?table',
     })
```

Reply:
And I made types of `space_name` consistent in a separate commit.

Reply:
When we consider index_id as a number we probably have more guarantees that spaces on different replicasets have the same index_id. And it's not so in case of spaces. In case of cartridge we could run some role with storage role on some replicaset we have different space_id with "plain" storage because "additional" role on the first storage could initialize its own spaces before storage spaces. Also there is local and temporary spaces that could be replicaset-specific and have own space_id.

I think space_id is not so much useful in distributed systems.

Reply:
Well, I agree with your concerns about index_id and space_id, so I reverted changes regarding index_id and space_id.

Reply:
Oleg's points convinced me too. Thanks!